src/components/icons/ExternalUserIcon.jsx

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+/**
+ * External user as SVG Icon
+ *
+ * @param {object} props  Data passed to the component
+ * @param {number} props.size  Width and height of the icon
+ * @param {string} props.className  Additional className for the component
+ * @param {object} props.style  Inline style object for the component
+ * @returns {object} JSX of the icon
+ *
+ */
+export default function ExternalUserIcon({size, className, style}) {
+  return (
+    <svg width={size} height={size} viewBox="0 0 20 20" fill="none" xmlns="http://www.w3.org/2000/svg" className={`wxc-icon ${className}`} style={style}>
+      <path d="M13.9063 14.0264V14.8436C13.9063 14.9679 13.9556 15.0872 14.0436 15.1751C14.1315 15.263 14.2507 15.3124 14.375 15.3124C14.4993 15.3124 14.6186 15.263 14.7065 15.1751C14.7944 15.0872 14.8438 14.9679 14.8438 14.8436V14.0264C14.8438 13.902 14.7944 13.7828 14.7065 13.6949C14.6186 13.607 14.4993 13.5576 14.375 13.5576C14.2507 13.5576 14.1315 13.607 14.0436 13.6949C13.9556 13.7828 13.9063 13.902 13.9063 14.0264Z" fill="var(--wxc-icon--external-user--color)" />
```

Comment:
instead of using that color, which is highly specific to the context, let's take the color from the surrounding (parent) element. This can be accomplished by using `currentColor` as the color name https://css-tricks.com/cascading-svg-fill-color/

You can list this color in a single place, on the `svg` element itself on line 16